Tests/StreamChatTests/Extensions/URL_EnrichedURL_Tests.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChat
+import XCTest
+
+final class URL_EnrichedURL_Tests: XCTestCase {
+    func test_schemeIsAdded_whenMissing() {
+        // GIVEN
+        let url = URL(string: "google.com")
+
+        // THEN
+        XCTAssertEqual(url?.enrichedURL.absoluteString, "http://google.com")
+    }
+
+    func test_urlWithSchemeIsNotChanged_whenHavingScheme() {
+        // GIVEN
+        let url = URL(string: "http://google.com")
```

Comment:
Let's update a bit to ensure the existed scheme is preserved
```suggestion
        let url = URL(string: "https://google.com")
```